src/config/field-rules.js

```diff
@@ -739,6 +756,23 @@ export default new Map([
                 type: 'HIDE_FIELD',
             }],
         },
+        {
+            field: 'notificationRecipient',
+            when: [{
+                field: 'notificationRecipient',
+                operator: 'EQUALS',
+                value: 'WEB_HOOK',
+            }],
+            operations: [{
+                field: 'deliveryChannels',
+                type: 'CHANGE_VALUE',
+                setValue: (model, fieldConfig) => {
+                    if(fieldConfig) {
+                        fieldConfig.value = model[fieldConfig.name] = ['HTTP']
+                    }
+                }
+            }],
+        },
```

Comment:
Duplicate rule, same as the one added above

Reply:
Yup, but they're under different models.

Reply:
Ah, oops, didn't notice. Yeah I wouldn't put that into a variable either

Reply:
Not easy to see from GitHub diffs, since that was truncated 😄